lib/note-list/index.tsx

```diff
@@ -59,244 +52,126 @@ type DispatchProps = {
 
 type Props = OwnProps & StateProps & DispatchProps;
 
-AutoSizer.displayName = 'AutoSizer';
-List.displayName = 'List';
-
-/**
- * Delay for preventing row height calculation thrashing
- *
- * this constant was determined experimentally
- * and is open to adjustment if it doesn't find
- * the proper balance between visual updates
- * and performance impacts recomputing row heights
- *
- * @type {Number} minimum number of ms between calls to recomputeRowHeights in virtual/list
- */
-const TYPING_DEBOUNCE_DELAY = 70;
-
-/**
- * Maximum delay when debouncing the row height calculation
- *
- * this is used to make sure that we don't endlessly delay
- * the row height recalculation in situations like when we
- * are constantly typing without pause. by setting this value
- * we can make sure that the updates don't happen
- * less-frequently than the number of ms set here
- *
- * @type {Number} maximum number of ms between calls when debouncing recomputeRowHeights in virtual/list
- */
-const TYPING_DEBOUNCE_MAX = 1000;
-
-/** @type {Number} height of title + vertical padding in list rows */
-const ROW_HEIGHT_BASE = 24 + 18;
-
-/** @type {Number} height of one row of preview text in list rows */
-const ROW_HEIGHT_LINE = 21;
-
-/** @type {Object.<String, Number>} maximum number of lines to display in list rows for display mode */
-const maxPreviewLines = {
-  comfy: 1,
-  condensed: 0,
-  expanded: 4,
-};
-
-/** @type {Number} height of a single header in list rows */
-const HEADER_HEIGHT = 28;
-
-/** @type {Number} height of a single tag result row in list rows */
-const TAG_ROW_HEIGHT = 40;
-
-/** @type {Number} height of a the empty "No Notes" div in the notes list */
-const EMPTY_DIV_HEIGHT = 200;
```

Comment:
Ah, we needed this in order to prevent the placeholder getting squished up against the top